cmd/aks-periscope/aks-periscope.go

```diff
@@ -56,16 +71,20 @@ func main() {
 			if err != nil {
 				log.Printf("Collector: %s, export data failed: %+v\n", c.GetName(), err)
 			}
+
 			waitgroup.Done()
 		}(c)
 	}
 
 	waitgroup.Wait()
 
 	diagnosers := []interfaces.Diagnoser{}
-	diagnosers = append(diagnosers, diagnoser.NewNetworkConfigDiagnoser(dnsCollector, kubeletCmdCollector, exporter))
 	diagnosers = append(diagnosers, diagnoser.NewNetworkOutboundDiagnoser(networkOutboundCollector, exporter))
 
+	if clusterType != "connectedcluster" {
```

Comment:
💡Note:  In line 49: https://github.com/Azure/aks-periscope/pull/51/files#diff-cdb15eaf022fc76984176a66a52183b8c3cb0710176cc9349519c5d43fb1d2ffR49 we check the condition with `equality` and at this line we check with `inequality` both intends to do the same check so why not keep it consistently.

Reply:
Sure. Will change it to be consistent.